data/genomics/homo_sapiens/demultiplexing/README.md

```diff
@@ -0,0 +1,4 @@
+This folder holds test datasets for modules (Vireo, Popscle, ScSplit, and Souporcell) that deconvolve single-cell multiplexed datasets. 
+The dataset, provided by Popscle (https://github.com/statgen/popscle/), contains genetic data from 2 donors. 
+For compatibility and testing within nf-core pipelines, the BAM and VCF files have been downsampled to include only data from chromosome 21.
+
```

Comment:
Is there any way that this dataset can be generated based on some of the data that already exists?

Reply:
Otherwise feel free to merge, I see you already downsampled it to the chromosome that is most compatible

Reply:
Thank you!